src/tools/spin/spinquic.cpp

```diff
@@ -519,6 +525,18 @@ void Spin(LockableVector<HQUIC>& Connections, std::vector<HQUIC>* Listeners = nu
             }
             break;
         }
+        case SpinQuicAPICallReceiveSetEnabled: {
+            auto Connection = Connections.TryGetRandom();
+            BAIL_ON_NULL_CONNECTION(Connection);
+            auto ctx = SpinQuicConnection::Get(Connection);
+            {
+                std::lock_guard<std::mutex> Lock(ctx->Lock);
+                auto Stream = ctx->TryGetStream();
+                if (Stream == nullptr) continue;
+                MsQuic->StreamReceiveSetEnabled(Stream, TRUE);
```

Comment:
Instead of always `TRUE` why not random and sometimes do `FALSE`?

Reply:
I'll make it false 50% of the time.